### PR TITLE
"Draft" item view should redirect to edit (with Draft banner), "Save for later" to /home

### PR DESCRIPTION
**Pull request by hobbitronics**
### Added
- ItemBanner to /[itemID]/edit.svelte
- $: status === 'Draft' && goToEditItem() to [itemID].svelte
- $$props.class to ItemBanner.svelte
### Changed
- onSaveForLater to redirect to /home
-
DraftItem

### CODE REVIEW

**Comment by merchako** [comment]
Can you put focus on the draft list item immediately after the redirect?

**Review by merchako — APPROVED** [review]

**Comment by hobbitronics** [comment]
> Can you put focus on the draft list item immediately after the redirect?

I will look into that. Made a card for now. https://trello.com/c/8T2eaYqr/127-focus-on-list-item-immediately-after-redirected-to-home-from-itemform

File: src/pages/items/[itemId].svelte [hunk 1]
```diff
@@ -53,6 +53,7 @@ $: items = $itemsByPolicyId[$user.policy_id] || []
 $: item = items.find((itm) => itm.id === itemId) || ({} as PolicyItem)
 $: itemName = item.name || ''
 $: status = (item.coverage_status || '') as ItemCoverageStatus
+$: status === 'Draft' && goToEditItem()
```
Comment (forevermatt) [inline src/pages/items/[itemId].svelte:56]: We probably only want to redirect to the edit page if this claim belongs to the current user, too

File: src/pages/items/[itemId]/edit.svelte [hunk 2]
```diff
@@ -5,6 +5,7 @@ import { loading } from '../../../components/progress'
 import { itemsByPolicyId, loadItems, PolicyItem, submitItem, updateItem } from '../../../data/items'
 import { goto } from '@roxi/routify'
 import { Page } from '@silintl/ui-components'
+import ItemBanner from '../../../components/banners/ItemBanner.svelte'
```
Comment (forevermatt) [inline src/pages/items/[itemId]/edit.svelte:8]: This could be moved up into the import from `components`

**Review by forevermatt — APPROVED** [review]

**Pull request merged**